WcaOnRails/app/assets/javascripts/competitions.js

```diff
@@ -27,10 +27,6 @@ onPage('competitions#edit, competitions#update, competitions#admin_edit, competi
     $('.competition_early_puzzle_submission_reason').toggle(this.checked);
   }).trigger('change');
 
-  $('input[name="competition[qualification_results]"]').on('change', function() {
-    $('.competition_qualification_results_reason').toggle(this.checked);
-  }).trigger('change');
-
```

Comment:
If the "qualification reason" is not coupled to the checkbox anymore, you should consider changing the description text below the text field to indicate that you need to fill this out if you want to use qualifications in the first place.

Reply:
Will do. One question, should I update this to be a new i18n key if I'm changing the description text? Or will translators be notified that the English text changed and that they should update the translated copy as well?

Reply:
Translators will be notified if the text of an existing key changes.
That's what we keep the `original_hash` in the translated files for ;)

Reply:
I don't think this was actually how I wanted this to work -- the text box is only supposed to show up when qualification times are used on the events page. Fixed it to work that way now, rather than updating the text.